docs/index.rst

```diff
@@ -41,3 +41,4 @@ Contents
     setup/historical_lsm_process.rst
     setup/forecast_framework.rst
     setup/web_application.rst
+    SPT REST API/SPT REST API.rst
```

Comment:
Can you rename SPT REST API to api or rest_api? It is better not not have spaces in the name. Thanks!